app/src/main/java/org/oppia/app/activity/InjectableAppCompatActivity.kt

```diff
@@ -1,17 +1,18 @@
 package org.oppia.app.activity
 
+import android.content.Intent
 import android.os.Bundle
 import android.os.PersistableBundle
 import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
+import org.oppia.app.R
 import org.oppia.app.application.OppiaApplication
 import org.oppia.app.fragment.FragmentComponent
-
 /**
  * An [AppCompatActivity] that facilitates field injection to child activities and constituent fragments that extend
  * [org.oppia.app.fragment.InjectableFragment].
  */
-abstract class InjectableAppCompatActivity: AppCompatActivity() {
+abstract class InjectableAppCompatActivity: AppCompatActivity()  {
```

Comment:
I think there is some extra white space just before `{`.

Reply:
resolved